test/cleanup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+readonly network_name="${NETWORK_NAME:-rcp-network}"
+readonly subnet_name="${SUBNET_NAME:-rcp-subnet}"
+readonly cluster_name="${CLUSTER_NAME:-rcp-cluster}"
+readonly zone="${ZONE:-us-west1-a}"
+
+# Suppress warning for listing 0 item.
+if [[ $(gcloud container clusters list --filter="name=${cluster_name} AND zone=${zone}" 2> /dev/null) ]]; then
+    gcloud container clusters delete ${cluster_name} --zone=${zone} --quiet
```

Comment:
why not just call delete directly instead of listing and have the error go to stderr?

```
gcloud container cluster delete "${cluster_name} ... || true
```

Reply:
Updated.